react/src/components/CreateMapModal/CreateMapModal.tsx

```diff
@@ -73,163 +123,150 @@ const CreateMapModal = ({
     });
   };
 
-  const handleSubmit = (values) => {
+  const handleSubmitCallback = () => {
     if (!userData) {
       setErrorMessage('User information is not available');
       return;
     }
+    const values = getValues();
     const projectData: ProjectRequest = {
       name: values.name,
       description: values.description,
-      system_file: values.system_file,
-      system_id: values.system_id,
+      system_file: values.systemFile,
+      system_id: values.systemId,
       system_path: `/${userData.username}`,
       watch_content: values.syncFolder,
       watch_users: values.syncFolder,
     };
     handleCreateProject(projectData);
   };
+
   return (
-    <Modal isOpen={isOpen} toggle={handleClose} size="xl">
-      <ModalHeader toggle={handleClose}>Create a New Map</ModalHeader>
-      <ModalBody>
-        <Section
-          bodyClassName="has-loaded-dashboard"
-          contentLayoutName={'twoColumn'}
-          // contentShouldScroll
-          className={`${styles['section']}`}
-          content={
-            <>
-              <SectionTableWrapper
-                className={`${styles['section-table-wrapper']}`}
+    <Modal
```

Comment:
I had my window taking up half my screen and the content on the bottom of the form side gets cut off. Also the file listing can disappear if the window isn't large enough. All follow on issues - just adjusting the how the modal flexs. Non-blockers for testing.

Reply:
Maybe splitter would be helpful in place of core-components Section?

Reply:
Yes good suggestions for follow on. I think that same issue is present in main